code/scripts/duplicate_string_find.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash  
+
+#grep -rhEo (recursive, no filename, extended regular expressions, prints only matching parts
+#excludes stable, only includes .hs files
+#cuts outer double quotations
+#first awk searches for instances that appear 3 or more times
+#second awk removes instances that have double quotations anywhere by the ends
+#last two seds add back the double quotations
```

Comment:
Note that, in theory a string like
```haskell
"some stuff and then \"an embedded quote\" is in there"
```
is actually different, because those escapes mean that it is a whole quote. Now, I'm pretty sure we don't have these (on purpose) except for somewhere in the printer to be able to output them. That would be worth checking.